src/factory/crawler-factory.js

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
I'm fine either way, but I can see the benefit of using camel case since the variable name can match up with the module name. I'll make that change.